conda/conda-build/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "legate-core" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ environ.get('GIT_DESCRIBE_TAG', environ.get('GIT_FULL_HASH', '')) }}
```

Comment:
Is there ever a situation where you wouldn't have a tag, but would have the hash?

Reply:
PEP386 compliance for version numbers, stash hash in the build string?

Reply:
Yeah, I changed this already. Will commit soon.